.github/workflows/beam_PostCommit_Java_PVR_Spark_Batch.yml

```diff
@@ -18,6 +18,8 @@
 name: PostCommit Java PVR Spark Batch
 
 on:
+  issue_comment:
```

Comment:
we've removed the issue_comment trigger for postcommits temporarily due to current approach is not scalable: #28909